runtime/src/main/java/org/corfudb/runtime/clients/AbstractClient.java

```diff
@@ -35,6 +38,17 @@
                 .setPriorityLevel(priorityLevel));
     }
 
+    <T> CompletableFuture<T> sendMessageWithFuture(CorfuMsg msg, Timer timer) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651457)